### PR TITLE
Do we need to elide summaries, or can we summarize them?

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Description
Uncomment if you want to provide a custom description.

- Remove the `elide` filter on summaries
- Try to shorten some real long summaries on services
- Looking at post summaries, I also tried to fix the blockquote summary layout

## Related Issue(s)
_Reminder to add related issue(s), if available._

- #534 

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

This was used in three places

- post lists
- work samples
- services

### CODE REVIEW

**Review by SondraE — APPROVED** [review]
@mirisuzanne I think this looks good. Some of the blog post summaries were longish, but I didn't see any summaries that were so long they looked broken. 

There's one blog post that has a quote as the summary, but in the list view there are no quotation marks. That looks a bit strange.
https://deploy-preview-535--oddleventy.netlify.app/2022/11/11/platform-tools/

I also see that some of the summaries have an orphan word. Is that okay or should we try to avoid it?

**Comment by mirisuzanne** [comment]
@SondraE I changed the blockquote to an inline quote, to make things simpler. How does that look?

File: content/_includes/post.macros.njk [hunk 1]
```diff
@@ -268,15 +268,8 @@ params:
           {{ talks.list(post | pageEvents, compact=true, show_links=false) }}
         </div>
       {% elif post.data.summary %}
-        {% set summary = post.data.summary | md %}
         <div class="p-summary">
-          {% if (card == 'large') %}
-            {{ summary | safe }}
-          {% else %}
-            <p>
-              {{ summary | elide | safe }}
```
[the remark this answers is not in the document]
Comment (jgerigmeyer) [reply content/_includes/post.macros.njk:277]: Or I'd even be tempted to say maybe we still want to elide post summaries, if not the others.

[the remark this answers is not in the document]
Comment (mirisuzanne) [reply content/_includes/post.macros.njk:277]: Yeah, removing some links would be good. But I still think: It's not a summary if we can't make it short enough to display.

[the remark this answers is not in the document]
Comment (mirisuzanne) [reply content/_includes/post.macros.njk:277]: (But @SondraE is suggesting we open a new card for dealing with posts, and I think that makes sense. Let's put the elide filter back on those for now?)

**Comment by SondraE** [comment]
@mirisuzanne 

> I changed the blockquote to an inline quote, to make things simpler. How does that look?

Much clearer!

**Review by jgerigmeyer — APPROVED** [review]

File: content/_includes/post.macros.njk (hunk 1 above)
[the remark this answers is not in the document]
Comment (jgerigmeyer) [reply content/_includes/post.macros.njk]: @mirisuzanne I added it back in for posts, but feel free to create a new issue to reconsider.

**Pull request merged**